$OpenBSD: patch-src_ml_glib_c,v 1.1 2013/02/01 10:22:01 dcoppa Exp $

Fix a segmentation fault due to a bad C cast
(upstream git commit 1c20cc7454f3b6f5341ef906fb0bf07ca681fc5e)

--- src/ml_glib.c.orig	Thu Aug 23 12:37:48 2012
+++ src/ml_glib.c	Fri Feb  1 09:47:10 2013
@@ -243,7 +243,7 @@ ML_2(g_log_set_fatal_mask, String_option_val, Int_val,
 
 CAMLprim value ml_g_log (value domain, value level, value msg)
 {
-  g_log (String_option_val(domain), Int_val(level), "%s", String_val(msg));
+  g_log (String_val(domain), Int_val(level), "%s", String_val(msg));
   return Val_unit;
 }
 
